.github/workflows/ci.yml

```diff
@@ -160,14 +160,11 @@ jobs:
 
       - name: Try extracting i18n strings (Python)
         if: success() || failure()
-        run: indico i18n extract-messages
+        run: indico i18n extract indico --python
 
       - name: Try extracting i18n strings (JS)
         if: success() || failure()
-        run: |
-          echo "::add-matcher::.github/matchers/react-jsx-i18n-problem-matcher.json"
-          FORCE_COLOR=1 npx react-jsx-i18n extract --ext jsx indico/web/client/ indico/modules/ > /dev/null
-          echo "::remove-matcher owner=react-jsx-i18n::"
+        run: indico i18n extract indico --javascript
```

Comment:
You need `--react` here; the JS extraction won't fail anyway, only the react one can fail if people do wrong stuff (e.g. dynamic content used directly inside a translatable block).